README.md

```diff
@@ -238,6 +260,28 @@ The **summary.tsv** output file generated by `staramr` contains the following co
 | SRR1952908 | aadA1, aadA2, blaTEM-57, cmlA1, gyrA (S83Y), sul3, tet(A) | streptomycin, ampicillin, chloramphenicol, ciprofloxacin I/R, nalidixic acid, sulfisoxazole, tetracycline |
 | SRR1952926 | blaTEM-57, gyrA (S83Y), tet(A)                            | ampicillin, ciprofloxacin I/R, nalidixic acid, tetracycline                                               |
 
+## detailed_summary.tsv
+
+The **detailed_summary.tsv** output file generated by `staramr` contains the following columns:
+
+* __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
+* __Gene__: The particular AMR gene detected from ResFinder, PlasmidFinder, and PointFinder.
```

Comment:
Can you remove **AMR** here too so it just says "The particular gene".

Reply:
Fixed this in commit 5e90065